content/docs/reference/authenticate-callback-path.mdx

```diff
@@ -65,12 +66,13 @@ See the [Kubernetes Deployment Reference](/docs/deploy/k8s/reference#authenticat
 ### Examples
 
 ```yaml
-# config file key
-authenticate_callback_path: "/custom/callback"
+authenticate_callback_path: '/custom/callback'
```

Comment:
I may have misunderstood your suggestion. I didn't move these under the tab because there are Core and Kubernetes examples, but no Enterprise example. I kept examples underneath/outside the tabs if only 2 out of 3 or all 3 tabs had examples. This setting has Core and K8s examples.

However, this K8s setting is for Core, so we could just include it under the K8s tab.